modules/library/main/src/main/java/org/geotools/filter/visitor/PostPreProcessFilterSplittingVisitor.java

```diff
@@ -424,7 +424,7 @@ public Object visit(PropertyIsNotEqualTo filter, Object notUsed) {
             return null;
         }
         
-	    private void visitBinaryComparisonOperator(BinaryComparisonOperator filter) {
+	    protected void visitBinaryComparisonOperator(BinaryComparisonOperator filter) {
```

Comment:
@ridethepenguin I am comfortable with leaving the tabs in gt-filter as your changes are so small and tab fixing would be quite messy.